DataScience/LogDownloader.py

```diff
@@ -176,7 +178,10 @@ def download_container(app_id, log_dir, container=None, conn_string=None, accoun
 
             try:
                 bp = bbs.get_blob_properties(container, blob.name)
-
+                # If total downloaded file size + current file size has exceeded max_size then stop downloading more
+                Logger.info('Total downloaded cooked log size in MB : {}\n'.format(total_download_size_MB))
```

Comment:
nit: Could this log be noisy? If I'm reading it right, it will log ~1000 times when we are downloading the max allowed size (since blobs are ~200-250MB).

Reply:
So the logs will look like below. I want to keep this because when there are lot of files and it keep downloading its hard to figure out how much of the work is done and how long it will take. This cumulative amount will give us a clue wrt that
Iterating through blobs...

Total downloaded cooked log size in MB : 0.0

Processing: 20201208234234/data/2020/12/14_0000000010.json (size: 25.611MB - Last modified: 2021-01-28 00:32:17+00:00)
Downloading with max_connections = 4...

Downloaded 25.611 MB in 3.1 sec. (8.347 MB/sec)

Total downloaded cooked log size in MB : 25.610791206359863

Stop downloading logs as total max cooked log size of 100 MB has been reached